.github/actions/deploy-release-candidate-v1/action.yml

```diff
@@ -0,0 +1,69 @@
+name: deploy-release-candidate
+description: Preparing a release candidate for QA
+
+inputs:
+  version:
+    description: A version of the release candidate e.g. "1.0.0"
+    required: true
+  revision-url:
+    description: A commit revision URL
+    required: true
+  owner:
+    description: An owner of the repository e.g. "dequelabs"
+    required: true
+  repo:
+    description: A repository name e.g. "jazzband"
+    required: true
+  slack-webhook:
+    description: A Slack channel webhook URL where the message will be sent
+    required: true
+  slack-channel:
+    description: A Slack channel name where the message will be sent
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: dequelabs/axe-api-team-public/.github/actions/get-release-issue-v1@main
+      id: get-release-issue
+      with:
+        version: ${{ inputs.version }}
+        owner: ${{ inputs.owner }}
+        repo: ${{ inputs.repo }}
+      # Check the release issue has been found
+    - name: Check the release issue
+      shell: bash
+      run: |
+        if [[ -z "${{ steps.get-release-issue.outputs.issue-url }}" ]]; then
+          echo "::error::No release issue has been found"
+          exit 1
+        fi
```

Comment:
I assume you're trying to exit early if no ticket is found? If so you should use the `gh run` commands to do that

```
gh run cancel ${{ github.run_id }}
gh run watch ${{ github.run_id }}
```

https://stackoverflow.com/a/75809743/2124254

Reply:
as discussed with @straker we leave the original code because it shows the correct error message